### ISSUE TITLE
VTK8 - mili zbuffer thrashing test failure

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#2342]
This is a bit of an odd test case but I think mili database is serving up a mesh with two surfaces that are defined by the same points. So, the surfaces fall on top of each other. Old VisIt would still draw this correctly. New VisIt manifests zbuffer thrashing.

Also seen in moab_02 test in serial mode and in parallel mode.

http://portal.nersc.gov/project/visit/tests/2018-05-14-22:00/surface_trunk_serial/databases_mili.html

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Mark Miller
Original creation: 05/02/2018 04:42 pm
Original update: 05/17/2018 01:56 pm
Ticket number: 3082
